### DISCUSSION TITLE
Using a devcontainer image as the base container for my CodeSpace environment

### DISCUSSION THREAD

**Discussion by chalmiller1** [devcontainers/community#75] [General]
Can someone help point me toward documentation on how to use a prebuilt devcontainer as the base image for my CodeSpace? Specifically, I need a devcontainer that provides me the ability to run docker compose in the base container. Thank you for the help!

**Comment by samruddhikhandale** [comment]
Hi 👋 

> Can someone help point me toward documentation on how to use a prebuilt devcontainer as the base image for my CodeSpace?

Have you looked at https://containers.dev/guide/prebuild ? It has more information on different types of prebuilds. 

> Specifically, I need a devcontainer that provides me the ability to run docker compose in the base container. Thank you for the help!

Dev Containers extensions and GitHub codespaces provides a set of templates which you can use. Have you tried using `Add Dev Container Configuration Files...` command? It lets you pick a pre-defined container configuration from a list.

Feel free to ask any other questions, thanks!
